azure/Guidelines.md

```diff
@@ -1292,6 +1292,52 @@ In addition to distributed tracing, Azure also uses a set of common correlation
 |x-ms-client-request-id       |Both      |Optional. Caller-specified value identifying the request, in the form of a GUID with no decoration such as curly braces (e.g. `x-ms-client-request-id: 9C4D50EE-2D56-4CD3-8152-34347DC9F2B0`). If the caller provides this header the service **must** include this in their log entries to facilitate correlation of log entries for a single request. Because this header can be client-generated, it should not be assumed to be unique by the service implementation.
 |x-ms-request-id              |Response  |Required. Service generated correlation id identifying the request, in the form of a GUID with no decoration such as curly braces. In contrast to the the `x-ms-client-request-id`, the service **must** ensure that this value is globally unique. Services should log this value with their traces to facilitate correlation of log entries for a single request.
```

Comment:
Again, I don't see why this must be a guid. A lot of people are using ULIDs instead of UUIDs now and we should allow a service to use whatever value is useful for the service. Other than this, I agree with the rest of this guideline.